### PR TITLE
bundle libraries.json

### PR DESCRIPTION
**Pull request by elharo**
@loosebazooka @chanseokoh

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -331,6 +331,13 @@
         </plugin>
       </plugins>
     </pluginManagement>
+
+    <resources>
```
Comment (loosebazooka) [inline pom.xml]: this can't go in src/main/resources?

Comment (elharo) [reply pom.xml]: It could. It doesn't have to.

Comment (loosebazooka) [reply pom.xml]: I think, as much as we can, we should follow the standard directory format for maven : https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html

Given that resources is already defined, can you move the file into `src/main/resources`?

Comment (loosebazooka) [reply pom.xml]: it would remove extra configuration from pom.xml.

**Pull request merged**